FunctionalPreprocessing/SubmitFunctionalPreprocessingPrereqHPC

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
As above, I think this belongs in a different branch, different review and merge later.